_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -34,14 +34,18 @@ acknowledgments:
 
 This rule applies to any [WAI-ARIA state or property][] that is specified on an [HTML or SVG element][namespaced element] that is [included in the accessibility tree][].
 
-## Expectation
+## Expectation 1
 
 For each test target, one of the following is true:
 
 - **global**: the test target is a [global state or property][global]; or
 - **semantic Role**: the test target is an [inherited][], [supported][], or [required][] [state][] or [property][] of the [semantic role][] of the element on which the test target is specified; or
 - **language feature**: the test target is specified on an [HTML element][namespaced element] and is allowed on that element. Which ARIA states or properties may be used on which element is described in [ARIA in HTML](https://w3c.github.io/html-aria/).
 
+## Expectation 2
+
+Each test target is not [prohibited][] on the [semantic role][] of the element on which it is specified.
```

Comment:
I find the negation inside the quantification a bit weird and feel it would flow better with
```suggestion
No test target is [prohibited][] on the [semantic role][] of the element on which it is specified.
```
but we may want to stick with the "each test target" formulation (even though there are a ew rules where it's tweaked a bit). So no strong opinion here.